telegram_bot.py

```diff
@@ -354,6 +363,70 @@ def _verify_profit_transfer_confirmation(self, update: Update, _: CallbackContex
                                       reply_markup=self._keyboards[self._keyboard_idx])
         return ConversationHandler.END
 
+    def _begin_set_notification(self, update: Update, _: CallbackContext) -> int:
+        reply_keyboard = [['entry_fill', 'close_fill'],
```

Comment:
I expect that the number of notification points will increase significantly in the future (position entry/position close/position reentry/stoploss reached...). For that reason I think it may be better to use an inline keyboard instead of a reply-keyboard? Could also be an evolutionary thing though :)

Reply:
I think you are planning many things in your mind, and it's really good things for everyone.
I agree with you :)